toolkit/offchain/src/update_governance/mod.rs

```diff
@@ -127,5 +153,9 @@ fn update_governance_tx(
 		inputs
 	});
 
+	for key in old_governance_authority.iter() {
```

Comment:
utilize `into_iter`, you own this vec and will not use it after that
```suggestion
	for key in old_governance_authority {
```